packages/first-boot-wizard/files/usr/lib/lua/firstbootwizard.lua

```diff
@@ -210,6 +218,14 @@ function fbw.apply_file_config(file, hostname)
     -- Clean previus lime configuration and replace lime-community
     config.reset_node_config()
     utils.execute("cp " .. filePath .. " /etc/config/" .. config.UCI_COMMUNITY_NAME)
+
+    -- Setup the shared lime-assets
+    local remote_hostname = string.sub(file, #fbw.COMMUNITY_HOST_CONFIG_PREFIX + 1)
+    local lime_community_assets_fname = lime_community_assets_name(remote_hostname)
+    if utils.file_exists(lime_community_assets_fname) then
+        utils.execute(string.format("tar xfz %s -C /etc/lime-assets/", lime_community_assets_fname))
```

Comment:
the use of string.format instead of the tradicional string concat might confuse the reader, as it is less used, more in these cases... not an issue per se, but I guess for readability might be better using the double dot expression

Reply:
Well I would say that string concatenation is not "traditional" at all :P 
But yes maybe it uses a too advanced feature?

Reply:
not too advance :) but felt that it was the thing that is used more often, and there is no specific use of the format method in this case, so I would go with the one more frequently used... 
this is an opinion though, so please do as you see fit :)